go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"vitess.io/vitess/go/mysql/sqlerror"
 	"vitess.io/vitess/go/protoutil"
+	"vitess.io/vitess/go/vt/vtctl/reparentutil"
```

Comment:
@GuptaManan100 Is this a lightweight package so that we don't end up including a large dependency chain in all places we add this to? That could increase potentially binary and / or compile times or makes parts harder to use as a library (if we do that for any of these bits really). 

In the past I've always checked this and also sometimes explicitly opted for a new package then just with the contants to mitigate problems mentioned above.